homeassistant/components/weather/__init__.py

```diff
@@ -26,6 +41,7 @@
 ATTR_FORECAST_WIND_BEARING = "wind_bearing"
 ATTR_FORECAST_WIND_SPEED = "wind_speed"
 ATTR_WEATHER_ATTRIBUTION = "attribution"
+ATTR_WEATHER_CONDITION = "condition"
```

Comment:
This constant doesn't represent a string in the home assistant API, since the current condition is the weather entity state itself. Wonder if we should add it anyway, since it can simplify the mapping, as you do?

Reply:
Good point, think I'll remove this.

Reply:
This has been removed.